code/datums/status_effects/gun_skill.dm

```diff
@@ -1,46 +1,77 @@
-/datum/status_effect/stacking/gun_skill
+/datum/status_effect/gun_skill
 	id = "gun_skill"
```

Comment:
why remove the stacking btw this was used to make kings scatter debuff decay

Reply:
Yeah I didn't like the decay. It's visually very unclear how badly debuffed you are, and once it gets under a certain number of stacks it functionally does nothing. I was probs going to reduce the total effect from offguard in a followup pr though.